ADA/ada.qmd

```diff
@@ -36,14 +35,65 @@ The ambition is to enable access to all datasets across DfE and ESFA and retire
 
 ---
 
+# What is Databricks?
+
+Databricks is a browser-based platform developed by Microsoft which allows you to use notebooks to access and analyse data. You can also use it for machine learning. Databricks notebooks support multiple programming languages, including R, Python, and SQL. Unlike RStudio or SQL Server Management Studio (SSMS), you don't have to download any software to be able to use it. You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can check the status of various datasets [in the data catalogue dashboard](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) 
```

Comment:
Would 'check the status of various datasets in the data catalogue dashboard' be more descriptive as the link text in the last sentence of this paragraph?